source/extensions/key_value/file_based/config.cc

```diff
@@ -36,6 +36,13 @@ void FileBasedKeyValueStore::flush() {
     file->write(it.first);
     file->write(absl::StrCat(it.second.length(), "\n"));
     file->write(it.second);
+    auto ttl_it = ttl_store_.find(it.first);
+    if (ttl_it != ttl_store_.end()) {
+      std::string ttl = std::to_string(ttl_it->second.count());
+      file->write("TTL");
```

Comment:
can we factor this out into a variable?  i also suggest something more obscure like `__KV_TTL__` just to minimize the chances of conflicting with the ttl key in the serialized file being part of the key in another.

Reply:
There isn't actually a risk of a conflict.  This file looks like `\n{len(key)}{key}\n{len(value)}{value}\nTTL{len(ttl)}{ttl}`

"TTL" goes before the length, so the only options are to see "TTL" or an integer representing length.

Reply:
ah you're right, makes sense, thanks!